[ADD] Appraisals: Appraisal templates (#13244)
odoo/documentation: content/applications/hr/appraisals/appraisal_templates.rst

===================
Appraisal templates
===================

> 

Reviewer: If including the table, be sure to include this short-hand for the 5 star icons
```suggestion

.. |5-stars| replace:: :icon:`fa-star-o` :icon:`fa-star-o` :icon:`fa-star-o` :icon:`fa-star-o`
                       :icon:`fa-star-o`
                       
```

appraisal templates, new templates :ref:`can be created <appraisals/create-template>`.

.. _appraisals/modify-template:

Modify appraisal templates
==========================

If needed, changes can be made to the default template. To view the default template, navigate to
:menuselection:`Appraisals app --> Configuration --> Appraisal Templates`.

The default template appears in a list view, named :guilabel:`Default Template`. Click on the
template to view the template details. Make any desired changes to the template.

The default template consists of the following questions:

+------------------------------------------------+-------------------------------------------------+
| Employee Feedback                              | Manager feedback                                |
+================================================+=================================================+
| | **My work**                                  | | **Feedback**                                  |
| | **What are my best achievement(s) since my   | | **Give one positive achievement that convinced|
|   last appraisal?**                            |   you of the employee's value.**                |

> | | *Describe something that made you proud, a   | | *List achievements that confirm the various   |
> |   piece of work positive for the company.*     |   job challenges.*                              |
> +------------------------------------------------+-------------------------------------------------+
> | | **What has been the most challenging aspect  | | **Evaluation**                                |
> |   of my work this past year and why?**         | | *Stress Resistance* :icon:`fa-star-o`         |
> | | *Did you face new difficulties? Did you      |   :icon:`fa-star-o` :icon:`fa-star-o`           |
> |   confront yourself to new obstacles?*         |   :icon:`fa-star-o`                             |
> |                                                | | *Time Management* :icon:`fa-star-o`           |
> |                                                |   :icon:`fa-star-o` :icon:`fa-star-o`           |
> |                                                |   :icon:`fa-star-o`                             |
> |                                                | | *Teamwork* :icon:`fa-star-o`                  |
> |                                                |   :icon:`fa-star-o` :icon:`fa-star-o`           |
> |                                                |   :icon:`fa-star-o`                             |
> |                                                | | *Autonomy* :icon:`fa-star-o`                  |
> |                                                |   :icon:`fa-star-o` :icon:`fa-star-o`           |
> |                                                |   :icon:`fa-star-o`                             |
> |                                                | | *Proactivity* :icon:`fa-star-o`               |
> |                                                |   :icon:`fa-star-o` :icon:`fa-star-o`           |
> |                                                |   :icon:`fa-star-o`                             |
> +------------------------------------------------+-------------------------------------------------+
> | | **What would I need to improve my work?**    | | **Improvements**                              |
> | | *How can the company help you with your need | | **How could the employee improve?**           |
> |   and objectives in order for you to reach     | | *From a manager's point of view, how could you|
> |   your goals and look for the best             |   help the employee to overcome their           |
> |   collaboration.*                              |   weaknesses?*                                  |
> +------------------------------------------------+-------------------------------------------------+
> | | **My future**                                | | **Short term (6-months) actions / decisions / |
> | | **What are my short and long-term goals      |   objectives**                                  |
> |   with the company, and for my career?**       | | *Do you need a rapid answer to the current    |
> | |  - *Give an example of short-term objective  |   situation?*                                   |
> |      (< 6 months)*                             |                                                 |
> | |  - *Give an example of long-term objective   |                                                 |
> |      (> 6 months)*                             |                                                 |
> +------------------------------------------------+-------------------------------------------------+
> | | **Which parts of my job do I most / least    | | **Long term (> 6 months) career discussion,   |
> |   enjoy?**                                     |   where does the employee want to go, how to    |
> | | *Every job has strong points, what are, in   |   help them reach this path?**                  |
> |   your opinion, the tasks that you enjoy the   | | *How do you see the employee in the future, do|
> |   most/the least?*                             |   your vision follow the employee's desire?*    |
> +------------------------------------------------+-------------------------------------------------+
> | | **My feelings**                              |                                                 |
> | | **How do I feel about the company...**       |                                                 |
> | |  - *Culture/Behavior:* :icon:`fa-star-o`     |                                                 |
> |      :icon:`fa-star-o` :icon:`fa-star-o`       |                                                 |
> |      :icon:`fa-star-o` :icon:`fa-star-o`       |                                                 |
> | |  - *Internal Communication:*                 |                                                 |
> |      :icon:`fa-star-o` :icon:`fa-star-o`       |                                                 |
> |      :icon:`fa-star-o` :icon:`fa-star-o`       |                                                 |
> |      :icon:`fa-star-o`                         |                                                 |
> | | **How do I feel about my own...**            |                                                 |
> | |  - *Job's content:* :icon:`fa-star-o`        |                                                 |
> |      :icon:`fa-star-o` :icon:`fa-star-o`       |                                                 |
> |      :icon:`fa-star-o` :icon:`fa-star-o`       |                                                 |
> | |  - *Work organization:* :icon:`fa-star-o`    |                                                 |
> |      :icon:`fa-star-o` :icon:`fa-star-o`       |                                                 |
> |      :icon:`fa-star-o` :icon:`fa-star-o`       |                                                 |
> | |  - *Remuneration:* :icon:`fa-star-o`         |                                                 |
> |      :icon:`fa-star-o` :icon:`fa-star-o`       |                                                 |
> |      :icon:`fa-star-o` :icon:`fa-star-o`       |                                                 |
> +------------------------------------------------+-------------------------------------------------+

Reviewer: I'm indifferent about including the default appraisal template as a table in the documentation. I lean towards no including it, since the instructions to find and modify the template in Odoo are sufficient.

If you choose to include it in the doc, I suggest we use a list table instead of the grid table as it will be easier to maintain, and we should also match the template text exactly. 

Here is the revised list table RST:

```suggestion
.. list-table:: Odoo **Appraisals** default appraisal template
   :header-rows: 1

   * - **Employee Feedback**
     - **Manager Feedback**

   * - **My work**

       **What are my best achievement(s) since my last appraisal?**

       *Describe something that made you proud, a piece of work positive for the company.*

       **What has been the most challenging aspect of my work this past year and why?**

       *Did you face new difficulties? Did you confront yourself to new obstacles?*

       **What would I need to improve my work?**

       *How can the company help you with your needs and objectives so you can reach your goals and
       foster better collaboration.*

       **My future**

       **What are my short and long-term goals with the company, and for my career?**

       - *Give an example of a short-term objective (< 6 months)*
       - *Give an example of a long-term objective (> 6 months)*

       **Which parts of my job do I most / least enjoy?**

       *Every job has strong points – which tasks do you enjoy the most and the least?*

       **My feelings**

       **How do I feel about the company...**

       - Culture/Behavior: |5-stars|
       - Internal Communication: |5-stars|

       **How do I feel about my own...**

       - Job content: |5-stars|
       - Work organization: |5-stars|
       - Remuneration: |5-stars|

     - **Feedback**

       **Give one positive achievement that convinced you of the employee's value.**

       *Some achievements that illustrate their strengths in addressing job challenges.*

       **Evaluation**

       +---------------------+-----------+
       | *Stress Resistance* | |5-stars| |
       +---------------------+-----------+
       | *Time Management*   | |5-stars| |
       +---------------------+-----------+
       | *Teamwork*          | |5-stars| |
       +---------------------+-----------+
       | *Autonomy*          | |5-stars| |
       +---------------------+-----------+
       | *Pro-activity*      | |5-stars| |
       +---------------------+-----------+

       **Improvements**

       **How could the employee improve?**

       *From a manager's point of view, how could you help them overcome their weaknesses?*

       **Short term (6-months) actions / decisions / objectives**

       *Do you need a rapid response to the current situation?*

       **Long term (> 6 months) career discussion, where does the employee want to go, how can you
       help them reach this path?**

       *How do you see the employee's future – does your vision align with their aspirations?*

```

appraisal templates, new templates :ref:`can be created <appraisals/create-template>`.

.. _appraisals/modify-template:

Modify appraisal templates
==========================

If needed, changes can be made to the default template. To view the default template, navigate to
:menuselection:`Appraisals app --> Configuration --> Appraisal Templates`.

The default template appears in a list view, named :guilabel:`Default Template`. Click on the
template to view the template details.

The default template consists of the following questions:

+------------------------------------------------+-------------------------------------------------+
| Employee Feedback                              | Manager feedback                                |
+================================================+=================================================+
| | **My work**                                  | | **Feedback**                                  |
| | **What are my best achievement(s) since my   | | **Give one positive achievement that convinced|
|   last appraisal?**                            |   you of the employee's value.**                |

> | | *Describe something that made you proud, a   | | *Some achievements comforting you in their    |
> |   piece of work positive for the company.*     |   strengths to face job's issues.*              |

Reviewer 2: ```suggestion
| | *Describe something that made you proud, a   | | *List achievements that confirm the employee’s strengths in     |
|   piece of work positive for the company.*     |   tackling job challenges.*              |
```

| | *Did you face new difficulties? Did you      |   :icon:`fa-star-o` :icon:`fa-star-o`           |
|   confront yourself to new obstacles?*         |   :icon:`fa-star-o`                             |
|                                                | | *Time Management* :icon:`fa-star-o`           |
|                                                |   :icon:`fa-star-o` :icon:`fa-star-o`           |
|                                                |   :icon:`fa-star-o`                             |
|                                                | | *Teamwork* :icon:`fa-star-o`                  |
|                                                |   :icon:`fa-star-o` :icon:`fa-star-o`           |
|                                                |   :icon:`fa-star-o`                             |
|                                                | | *Autonomy* :icon:`fa-star-o`                  |
|                                                |   :icon:`fa-star-o` :icon:`fa-star-o`           |
|                                                |   :icon:`fa-star-o`                             |

> |                                                | | *Pro-activity* :icon:`fa-star-o`              |

Reviewer 2: ```suggestion
|                                                | | *Proactivity* :icon:`fa-star-o`              |
```

|                                                |   :icon:`fa-star-o` :icon:`fa-star-o`           |
|                                                |   :icon:`fa-star-o`                             |
+------------------------------------------------+-------------------------------------------------+
| | **What would I need to improve my work?**    | | **Improvements**                              |
| | *How can the company help you with your need | | **How could the employee improve?**           |

> |   and objectives in order for you to reach     | | *From a manager point of view, how could you  |

Reviewer 2: ```suggestion
|   and objectives in order for you to reach     | | *From a manager's point of view, how could you  |
```

|   collaboration.*                              |   weaknesses?*                                  |
+------------------------------------------------+-------------------------------------------------+
| | **My future**                                | | **Short term (6-months) actions / decisions / |
| | **What are my short and long-term goals      |   objectives**                                  |
|   with the company, and for my career?**       | | *Do you need rapid answer to the current      |
| |  - *Give an example of short-term objective  |   situation?*                                   |
|      (< 6 months)*                             |                                                 |
| |  - *Give an example of long-term objective   |                                                 |
|      (> 6 months)*                             |                                                 |
+------------------------------------------------+-------------------------------------------------+
| | **Which parts of my job do I most / least    | | **Long term (> 6 months) career discussion,   |

> |   enjoy?**                                     |   where does the employee wants to go, how to   |

Reviewer 2: ```suggestion
|   enjoy?**                                     |   where does the employee want to go, how to   |
```

| | **How do I feel about my own...**            |                                                 |
| |  - *Job's content:* :icon:`fa-star-o`        |                                                 |
|      :icon:`fa-star-o` :icon:`fa-star-o`       |                                                 |
|      :icon:`fa-star-o` :icon:`fa-star-o`       |                                                 |
| |  - *Work organization:* :icon:`fa-star-o`    |                                                 |
|      :icon:`fa-star-o` :icon:`fa-star-o`       |                                                 |
|      :icon:`fa-star-o` :icon:`fa-star-o`       |                                                 |
| |  - *Remuneration:* :icon:`fa-star-o`         |                                                 |
|      :icon:`fa-star-o` :icon:`fa-star-o`       |                                                 |
|      :icon:`fa-star-o` :icon:`fa-star-o`       |                                                 |
+------------------------------------------------+-------------------------------------------------+

> Make any desired changes to the template.

Reviewer 2: this line kinda gets lost between the table and the next heading. Let's either move it up to before the table or remove it entirely (:

.. _appraisals/create-template:

Create appraisal templates
==========================

Large companies with many departments may prefer department-specific appraisal templates, instead of
using a default universal template. Creating and using department-specific templates can be helpful
if specific feedback is needed.

.. example::
   An appliance repair company has two main types of employees, office workers who handle
   administrative tasks and scheduling, and repair technicians who work in the field, performing

>    repairs at customer homes.

Reviewer 2: ```suggestion
   repairs at customers' homes.
```
